### PR TITLE
Task without changing and deploy

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**

### CODE REVIEW

**Comment by YevhenOstrovskiy** [comment]
[DEMO LINK](https://YevhenOstrovskiy.github.io/react_todo-app-with-api/)

**Review by l0ad1nggg — CHANGES_REQUESTED** [review]
Looks great, but tests should pass(

File: src/components/TodoItem/TodoItem.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+import { Todo } from '../../types/Todo';
+
+// eslint-disable-next-line import/no-cycle
+import {
+  LoadingContext,
+  TodoUpdateContext,
+} from '../../TodosContext/TodosContext';
+
+type Props = {
```
Comment (DanilWeda) [inline src/components/TodoItem/TodoItem.tsx:19]: use interface prefer. 
Types have unique features but the interface is best for scaling.

File: src/components/TodoItem/TodoItem.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            className="todo__title-field"
+            placeholder="Empty todo will be deleted"
+            value={title}
+            onChange={(event) => setTitle(event.target.value)}
+            onBlur={handleSubmit}
+            onKeyUp={(event) => {
+              if (event.key === 'Escape') {
+                setChangeTitle(false);
+                setTitle(todo.title);
+                console.log('escape was pushed');
```
Comment (DanilWeda) [inline src/components/TodoItem/TodoItem.tsx:128]: remove console.
write handler for it

File: src/components/TodoList/TodoList.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { TodoItem } from '../TodoItem/TodoItem';
+import { Status } from '../../types/Status';
+import { filterByStatus } from '../../services/filterByStatus';
+
+// eslint-disable-next-line import/no-cycle
+import {
+  TodosContext,
+} from '../../TodosContext/TodosContext';
+
+type Props = {
```
Comment (DanilWeda) [inline src/components/TodoList/TodoList.tsx:13]: use interface prefer. 
Types have unique features but the interface is best for scaling.

**Review by DanilWeda — APPROVED** [review]
Good job. 
Approved! 
Thanks!

**Pull request closed without merging**